catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DuplicateCommands.java

```diff
@@ -219,12 +271,15 @@ protected void duplicateInBatches(
    * available in the ingestFacade, the query should only return the remaining wanted metacards, not
    * the full batch size.
    *
-   * @param totalPossible - the total hits from the ingestFacade or the maxMetacards
-   * @param index - the index to be queried next
-   * @return how many metacards should be returned by a query starting at {@param index}
+   * @param maxMetacards - maximum number of metacards wanted or 0
```

Comment:
❓In what situation would `maxMetacards` be 0?

Reply:
If it is unspecified